Library/Homebrew/test/language/java_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "language/java"
+
+describe Language::Java do
+  describe "#self.java_home_env" do
+    it "returns java_home path with version if version specified" do
+      java_home = described_class.java_home_env("blah")
+      expect(java_home[:JAVA_HOME]).to include(" --version blah")
```

Comment:
I put the space there because the function itself had this space: https://github.com/mansimarkaur/brew/blob/690e841fcac1088043e259e273d79707e0616992/Library/Homebrew/language/java.rb#L4